geniza/footnotes/models.py

```diff
@@ -522,14 +560,19 @@ def content_html(self):
     @cached_property
     def content_html_str(self):
         "content as a single string of html, if available"
-        # content html is a dict; values are lists of html content
-        return "\n".join(
-            [
-                section
-                for canvas_annos in self.content_html.values()
-                for section in canvas_annos
-            ]
+        # parse to add line numbers
+        parser = HTMLLineNumberParser()
+        parser.feed(
+            "\n".join(
+                [
+                    section
+                    # content html is a dict; values are lists of html content
+                    for canvas_annos in self.content_html.values()
+                    for section in canvas_annos
+                ]
+            )
```

Comment:
I think we only want the line numbers for solr index data, not other use cases (like my github export), so I vote not applying it here by default.  What about a utility method that applies the parser and generates the output, so it's a one-line call? maybe something like `explicit_line_numbers`